plugin/src/main/kotlin/com/jetbrains/packagesearch/plugin/ui/panels/side/HeaderAttributesTab.kt

```diff
@@ -0,0 +1,293 @@
+                stiffness = Spring.StiffnessVeryLow
+            )
+        )
+    }
+}
+
+@Composable
+fun AttributeItem(modifier: Modifier = Modifier, attributeName: String, nestedAttributesName: List<String>) {
+
+    Row(modifier = modifier, verticalAlignment = Alignment.Top, horizontalArrangement = Arrangement.spacedBy(8.dp)) {
+        Row(
+            modifier = Modifier.width(160.dp),
+            verticalAlignment = Alignment.Bottom,
+            horizontalArrangement = Arrangement.spacedBy(4.dp)
+        ) {
+            Text(text = attributeName)
+            LabelInfo(nestedAttributesName.size.toString())
+        }
+
+        Text(text = nestedAttributesName.joinToString("\n"))
+    }
+}
+
+
+@Preview
+@Composable
+private fun HeaderAttributesPreviewTab() {
+    val activeTabMock = InfoPanelContent.Attributes.Declared(
+        tabTitle = "Declared",
+        variantName = "jvm",
+        attributes = generateAttributesMock()
+    )
+    val scrollStateMock = ScrollState(0)
+    Column(Modifier.padding(8.dp)) {
+        IntUiTheme(true) {
+            Column(Modifier.background(LocalGlobalColors.current.paneBackground).padding(16.dp)) {
+                HeaderAttributesTabImpl(
+                    content = activeTabMock,
+                    scrollState = scrollStateMock,
+                    contentTitle = "Search results that support:",
+                    attributeTypeName = "Platforms:",
+                    sourceSetString = "Source Sets:"
+                )
+
+            }
+
+            Divider(orientation = Orientation.Horizontal, modifier = Modifier.padding(vertical = 16.dp))
+
+        }
+
+
+        IntUiTheme(false) {
+            Column(Modifier.background(LocalGlobalColors.current.paneBackground).padding(16.dp)) {
+                HeaderAttributesTabImpl(
+                    content = activeTabMock,
+                    scrollState = scrollStateMock,
+                    contentTitle = "Search results that support:",
+                    attributeTypeName = "Platforms:",
+                    sourceSetString = "Source Sets:"
+                )
+
+            }
+        }
+    }
+}
+
+private fun generateAttributesMock(): List<PackageSearchModuleVariant.Attribute> {
+    val attributes = mutableListOf<PackageSearchModuleVariant.Attribute>()
+
+    platformListMock.take(Random.nextInt(2, 10)).forEach {
+        attributes.add(
+            if (Random.nextBoolean()) {
+                PackageSearchModuleVariant.Attribute.NestedAttribute(
+                    it,
+                    platformListMock.take(Random.nextInt(2, 10)).map {
+                        PackageSearchModuleVariant.Attribute.StringAttribute(it)
+                    }
+                )
+
+
+            } else {
+
+                PackageSearchModuleVariant.Attribute.StringAttribute(it)
+            }
+        )
+    }
+
+    return attributes
+
+}
+
+internal val platformListMock = buildList {
```

Comment:
We do not have any treeshaking step, as such those top level variable will stay in memory in the final application. We should move all preview related stuff in a separated module, but for now, just make top level attributes a function with `get() = `.